### PR TITLE
Correct full and right footpad switch state

### PR DESCRIPTION
**Pull request by Defying**

### CODE REVIEW

File: float_accessories/lib/can.lisp [hunk 1]
```diff
@@ -260,8 +260,18 @@
                                 ;(def beep-reason-t (shr switch-state-byte 4))
                                 (setq handtest-mode (= (bitwise-and switch-state-byte 0x08) 0x08))
 
-                                ;(def footpad-adc1-t (bufget-u8 data 11))
-                                ;(def footpad-adc2-t (bufget-u8 data 12))
+                                (def footpad-adc1-t (/ (bufget-u8 data 11) 50))
```
Comment (contactsimonwilson) [inline float_accessories/lib/can.lisp:263]: These need to be made floats before the conversion
```suggestion
                                (def footpad-adc1-t (/ (to-float (bufget-u8 data 11)) 50))
```

**Review by contactsimonwilson — CHANGES_REQUESTED** [review]

**Comment by Relys** [comment]
Had been fixed internally. Just pushed large commit.

**Comment by contactsimonwilson** [comment]
@Relys isn't the fix for the ADC voltages still missing from the latest commit?

**Comment by Relys** [comment]
Implemented

**Pull request closed without merging**